.github/workflows/manual-deploy.yml

```diff
@@ -48,8 +59,5 @@ jobs:
       - name: Deploy Files
         run: |
           echo "$S3_BUCKET"
-          aws s3 cp pinger.js $S3_BUCKET/pinger.js
-          aws s3 cp favicon.ico $S3_BUCKET/favicon.ico
-          aws s3 cp index.html $S3_BUCKET/index.html
-          aws s3 cp styles.css $S3_BUCKET/styles.css
-          aws s3 cp urls.json $S3_BUCKET/urls.json
\ No newline at end of file
+          aws s3 sync ./out $S3_BUCKET --recursive
```

Comment:
@wenjunche check me please!

Reply:
don't think ever used sync,  but seems to be much better than cp